### PR TITLE
qol: sect contracts now have a help button that explains how contracts work

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

It's just a button in the top right that you can hover over to read a tooltip

## Why It's Good For The Game

This is some feedback I got from live servers

## Changelog

:cl:
qol: sparring sect now has an info button in the top right of the contract menu
/:cl:

### CODE REVIEW

File: tgui/packages/tgui/interfaces/SparringContract.tsx [hunk 1]
```diff
@@ -70,8 +70,27 @@ export const SparringContract = (props, context) => {
...
+                        The Chaplain's Deity wishes for honorable fighting.
+                        As such, it uses contracts. Signing your name will
+                        set the terms for the battle. Then, the person you
+                        intend to spar with must sign the other side. If terms
+                        are changed on an already signed contract, the
+                        signatures will erase and the new terms must be
+                        re-agreed upon.
+                        `}
+                        icon="info">
+                        Contract?
```
Comment (Watermelon914) [inline tgui/packages/tgui/interfaces/SparringContract.tsx:90]: might just want to have it as an info icon

Comment (LemonInTheDark) [reply tgui/packages/tgui/interfaces/SparringContract.tsx:90]: @tralezab ?

**Review by stylemistake — APPROVED** [review]

File: tgui/packages/tgui/interfaces/SparringContract.tsx (hunk 1 above)
Comment (tralezab) [reply tgui/packages/tgui/interfaces/SparringContract.tsx:90]: i think it's a bit better with contract, just a style choice here

Comment (Watermelon914) [reply tgui/packages/tgui/interfaces/SparringContract.tsx:90]: could you post pictures of what it looks like? I'm thinking it'll just look weird with both the icon and the text.

**Pull request merged**